README.md

```diff
@@ -1,14 +1,59 @@
 dans-dbf-lib
 ============
 
-DANS DBF Library - Read and write DBF database files.
+DANS DBF Library - Java classes to read, write and update DBF database files.
 
 DESCRIPTION
 ===========
 
-Java library to read and write the contents of old [DBF] \(dBase) databases. It was produced for the [MIXED] project.
+DANS DBF is a Java library for reading, writing and updating
+the contents of old [DBF] \(dBase) databases.
+It was produced for the [MIXED] project.
 It is no longer maintained. You are however free to fork it, if you wish. 
 
+DANS DBF requires Java 1.5, or later, and has no dependencies.
+
+The following program demonstrates how the library is used.
+
+```java
+import nl.knaw.dans.common.dbflib.*;
+
+import java.io.File;
+import java.util.Iterator;
+import java.util.List;
+
```

Comment:
Nice example code! It could still be improved a bit though, by making it more concise:
* Remove the try-catch, as it doesn't add any functionality. (Just add `throws Exception` to main, if this proves necessary to compile the code`). The default Java behaviour will be to print a stacktrace if some error isn't caught.
* Maybe you should use the project-relative path to `cars.dbf` instead of `/tmp`: 
  `new File("src/test/resources/dbase3plus/cars/cars.db")`, that way people won't have to first look for the dbf file.

Please, take these improvements under consideration. Thanks again for contributing!